broadinstitute/scp-rails-baseimage
# Updating to ubuntu2004:latest base image (SCP-3972)

This update incorporates the Google managed `ubuntu2004:latest` base image, as well as updating to the latest releases of the `phusion/baseimage-docker` and `phusion/passenger-docker` images.  This will allow us to update to Ruby `3.1` if so desired, as well as the latest versions of `nginx` and `passenger`.  It also contains some small updates to how images are tagged that are the result of Phusion changing how images are tagged when built directly from their build scripts.

MANUAL TESTING

Our existing Jenkins automation has shown that using the new candidate image, we can build a new SCP image and run nginx: https://scp-jenkins.dsp-techops.broadinstitute.org/job/scp-rails-baseimage-test/267/console

If you want to verify building the candidate image, you can pull this branch and run `ci/build`.  This will take ~15-20m to complete.  Note: you will need to be logged into `vault`, and `docker` must be running.
